extensions/wikia/AdEngine/js/provider/directGpt.js

```diff
@@ -52,10 +52,6 @@ define('ext.wikia.adEngine.provider.directGpt', [
 		{
 			afterSuccess: function (slotName) {
 				slotTweaker.removeDefaultHeight(slotName);
-				if (!uapContext.isBfaaLoaded()) {
```

Comment:
`uapContext` is unused now

Reply:
I keep forgetting that. Removed from code and tests.